tools/crm_simulate.c

```diff
@@ -884,9 +884,6 @@ main(int argc, char **argv)
     }
 
     if (args->verbosity > 0) {
-        /* Redirect stderr to stdout so we can grep the output */
-        close(STDERR_FILENO);
-        dup2(STDOUT_FILENO, STDERR_FILENO);
```

Comment:
After seeing it, I've changed my mind :)

Instead of removing it entirely, let's guard it with PCMK__COMPAT_2_0 so anyone building with --enable-compat-2.0 keeps the old behavior.

Reply:
Fixed.